From a30b8dbe9cc199067c0cc5a2dc0d7322c9f4303a Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Thu, 10 Nov 2011 15:06:26 +0800
Subject: [PATCH 389/691] ENGR00161860 Fix build issue when enable NAND in
 Uboot

Add the MACRO define CONFIG_APBH_DMA which missed in last commit

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 include/configs/mx6q_arm2.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/include/configs/mx6q_arm2.h b/include/configs/mx6q_arm2.h
index e21bfad..0a9ff07 100644
--- a/include/configs/mx6q_arm2.h
+++ b/include/configs/mx6q_arm2.h
@@ -259,6 +259,7 @@
 	#define CONFIG_CMD_EXT2		1
 
 	/* NAND is the unique module invoke APBH-DMA */
+	#define CONFIG_APBH_DMA
 	#define CONFIG_APBH_DMA_V2
 	#define CONFIG_MXS_DMA_REG_BASE	ABPHDMA_BASE_ADDR
 #endif
-- 
1.7.7.4

